packages/expo-image-loader/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "expo-image-loader",
+  "version": "1.0.0",
+  "description": "Image loader",
+  "main": "index.js",
+  "keywords": [
+    "unimodules",
+    "image-loader",
+    "expo-image-loader"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git",
+    "directory": "packages/expo-image-loader"
+  },
+  "bugs": {
+    "url": "https://github.com/expo/expo/issues"
+  },
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "homepage": "https://github.com/expo/expo/tree/master/packages/expo-image-loader"
```

Comment:
Shouldn't we also add `peerDependencies` for:
- `react-native`, since we depend on `React-Core` pod, so not only this module depends on React, but on it's specific version (>= 0.60?) [adapter example](https://github.com/expo/expo/blob/0bfe783584a4072bfe42471289472c3be65a2ec1/packages/@unimodules/react-native-adapter/package.json#L36)

and `unimodulePeerDependencies` for:

- `unimodules-image-loader-interface`, since we implement it and assume it will be available (and in specific version, >= current, otherwise we wouldn't implement the whole interface, just the `forManipulation` method)
- `@unimodules/core`, since we use unimodules' stuff